### PR TITLE
coin_value function

### PR DESCRIPTION
**Pull request by MCarlomagno**
This function compares the value of the provided symbol against usd. Currently supporting:

| Symbol | CoinGecko Id | name |
|---------|--------------|-------|
|bnb	        |binancecoin        | Binance Smart Chain|
|avax	|avalanche-2      | Avalanche |
|matic	|matic-network | Polygon |
|rbtc         |rootstock          | Rootstock |

[Example endpoint with "bnb" symbol](https://n6e63vpxh2.execute-api.us-east-1.amazonaws.com/production/beacons/coin_value?symbol=bnb)

#### Example result
`{"symbol":"bnb", "value":213.74, "coingecko_api_name":"binancecoin"}`

### CODE REVIEW

**Comment by MCarlomagno** [comment]
### Coin value refresh

[source code](https://github.com/api3dao/beacons-api/blob/coin-value-vs-usd/src/coin-value-refresh.ts)

Cron function that runs every 10 minutes.
1.  Selects each existing `coin_gecko_id` from `coin_value` table.
2. Fetchs the current value for each one using CoinGecko API.
3. Updates the `value` and `updated_at` column for each row.

**Review by aquarat — APPROVED** [review]
This looks great.
We may want to abstract the 
```
{ totalTimeoutMs: 5_000, retries: 2 }
```
configuration for promise-utils into a constant in `constants.ts`.

We'd like something extra, which I'll describe separately. That can be implemented in a new PR.

Thanks for the good work :)

File: src/coin-value-refresh.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+  const operation = async () =>
+    db.query(
+      `
+			update coin_value as current set
+			value = updated.value, updated_at = current_timestamp
+			from (select unnest($1::text[]) as symbol, unnest($2::double precision[]) as value)
+			as updated(symbol, value)
+			where current.symbol = updated.symbol;
+			`,
```
Comment (aquarat) [inline src/coin-value-refresh.ts:76]:
```suggestion
      `
			UPDATE coin_value AS current 
			SET
			  value = updated.value, updated_at = current_timestamp
			FROM 
			  (select unnest($1::text[]) as symbol, unnest($2::double precision[]) as value)
			AS updated(symbol, value)
			WHERE current.symbol = updated.symbol;
			`,
```
It's a nit, but makes things easier to read and looks neat: can you make the operator symbols upper case in future?

Separately, the way in which you've structured this update is very interesting @ FROM clause.

**Comment by MCarlomagno** [comment]
Thanks for all the valuable feedback 😃 all the suggestions were added.

**Review by aquarat — APPROVED** [review]
LGTM :fire: 
Thank you :)

**Pull request merged**